frontend/src/libs/types/types.ts

```diff
@@ -5,3 +5,11 @@ export {
   type ValidationSchema,
   type ValueOf,
 } from 'shared/build/index.js';
+
+type Option = {
+  value: string;
```

Comment:
Option.value may not always be a `string`
And move it's declaration out of root export file

Reply:
what types must be on values?